File: NopTHAndroid/Cau2_temperature_Conversion/app/src/main/java/ntu/thu_62132086/cau2_temperature_conversion/MainActivity.java
package ntu.thu_62132086.cau2_temperature_conversion;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final EditText NumberC = findViewById(R.id.editC);
        final EditText NumberF = findViewById(R.id.editF);
        Button C = findViewById(R.id.CtoF);
        Button F = findViewById(R.id.FtoC);
        Button clear = findViewById(R.id.clear);
        C.setOnClickListener(v -> {
            double f = Double.parseDouble(NumberC.getText().toString());
            Double c = (f*9)/5+32;
            NumberF.setText(String.valueOf(c));
        });
        F.setOnClickListener(v -> {
            double c = Double.parseDouble(NumberF.getText().toString());
            Double f = (c-32)*5/9;
            NumberC.setText(String.valueOf(f));
        });
        clear.setOnClickListener(view -> {
            NumberF.setText("");
            NumberC.setText("");
        });
    }
}

File: NopTHAndroid/BaiTapTongHop/app/src/main/java/ntu/thu_62132086/baitaptonghop/MainActivity.kt
package ntu.thu_62132086.baitaptonghop

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        // Default Fragment
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragment_container, WelcomeFragment())
            .commit()
        val bottomNav = findViewById<BottomNavigationView>(R.id.bottomNavigationView)
        bottomNav.setOnNavigationItemSelectedListener { item ->
            var selectedFragment: Fragment? = null
            when (item.itemId) {
                R.id.navigation_welcome -> selectedFragment = WelcomeFragment()
                R.id.navigation_cau2 -> selectedFragment = Cau2Fragment()
                R.id.navigation_cau3 -> selectedFragment = Cau3Fragment()
                R.id.navigation_cau4 -> selectedFragment = Cau4Fragment()
            }
            // It will help to replace the
            // one fragment to other.
            if (selectedFragment != null) {
                supportFragmentManager.beginTransaction()
                    .replace(R.id.fragment_container, selectedFragment).commit()
            }
            return@setOnNavigationItemSelectedListener true
        }
    }
}